tests/core/test_algorithm.py

```diff
@@ -23,4 +13,11 @@ def test_nnls(rng):
     x_glass = nnls_glass(a, b)
     x_scipy, _ = nnls_scipy(a, b)
 
-    np.testing.assert_allclose(x_glass, x_scipy)
+    assert np.allclose(x_glass, x_scipy)
```

Comment:
Why remove `numpy.teting`? It is more powerful. `ruff` in general warns against `assert` https://docs.astral.sh/ruff/rules/assert

Reply:
Ah, I did not know this. TIL, thanks!